Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -3206,8 +3216,8 @@ - (void)yapDatabaseModified:(NSNotification *)notification
         [self setNavigationTitle];
     }
 
-    if (!
-        [[self.uiDatabaseConnection ext:TSMessageDatabaseViewExtensionName] hasChangesForNotifications:notifications]) {
+    if (![[self.uiDatabaseConnection ext:TSMessageDatabaseViewExtensionName] hasChangesForGroup:self.thread.uniqueId
+                                                                                inNotifications:notifications]) {
```

Comment:
We only care about changes in this thread. This should allow the view to ignore changes from incoming messages for other threads.